source/fundamentals/connection.txt

```diff
@@ -55,10 +58,31 @@ options. In the example, we set two connection options:
 ``maxPoolSize=20`` and ``w=majority``. To learn more about connection
 options, read the :ref:`golang-connection-options` section of this guide.
 
-The following code shows how you can use a connection string in a
-client to connect to MongoDB. In this example, the client uses 
-the :ref:`Stable API <golang-stable-api>` feature, though
-you can connect to MongoDB without this feature.
+Connection Example
+~~~~~~~~~~~~~~~~~~
+
+To connect to MongoDB, you need to create a client. A client manages
+your connections and runs database commands.
+
+You can create a client by calling the ``Connect()`` method. You can
+specify your connection string and other client options by using the
+``options`` package. You can specify your connection URI by passing it
+to the ``ApplyURI()`` method. You can set any other options by using
+methods from the options package.
```

Comment:
Suggestion:

The last two sentences seem to have some redundancies with the second sentence, so I think it can be made more concise. Also, keep consistent with the formatting for the "options package" so that it's less likely to get confused with "[client] options".

E.g.

You can pass your connection string and other client options to the Connect() method using methods in the options package. To specify your connection URI, pass it to the ApplyURI() method. To specify other options, call the appropriate helper method from the options package. (note that you should apply monospace where appropriate)